datafusion/sqllogictest/test_files/array.slt

```diff
@@ -4972,11 +4972,19 @@ NULL 1 1
 
 ## array_has/array_has_all/array_has_any
 
-query BB
+# If lhs is empty, return false
+query B
+select array_has([], 1);
+----
+false
+
+# If rhs is Null, we returns Null
+query BBB
 select array_has([], null),
-       array_has([1, 2, 3], null);
+       array_has([1, 2, 3], null),
+       array_has([null, 1], null);
 ----
-false false
+NULL NULL NULL
 
 #TODO: array_has_all and array_has_any cannot handle NULL
```

Comment:
Is there coverage in this file for `array_has` when the first argument is an actual column (not a scalar)? I didn't see such tests but maybe I missed them

Reply:
Yes, there is.

Reply:
https://github.com/apache/datafusion/blob/a58416c2eae47d19f0ef4756768f1e978d7c7dd3/datafusion/sqllogictest/test_files/array.slt#L5041-L5057